tests/opwen_email_server/integration/test_celery.py

```diff
@@ -3,38 +3,80 @@
 from string import ascii_letters
 from unittest import TestCase
 from unittest import skipUnless
+import ast
+import inspect
 
 from cached_property import cached_property
 from kombu import Connection
 from kombu import Exchange
 from kombu import Queue
 
 from opwen_email_server.config import QUEUE_BROKER
+import opwen_email_server.integration.celery as celery
 
 
 class TransportTests(TestCase):
-    exchange_name = getenv('KOMBU_EXCHANGE', 'testexchange')
-    queue_name = getenv('KOMBU_QUEUE', 'testqueue')
-    routing_key = getenv('KOMBU_ROUTING_KEY', 'testkey')
-
-    @cached_property
-    def exchange(self) -> Exchange:
-        return Exchange(self.exchange_name, 'direct', durable=True)
-
-    @cached_property
-    def queue(self) -> Queue:
-        return Queue(self.queue_name, exchange=self.exchange, routing_key=self.routing_key)
-
-    @skipUnless(QUEUE_BROKER, 'no celery broker configured')
-    def test_send_message(self):
-        random_message = ''.join(choice(ascii_letters)  # nosec
-                                 for _ in range(30))
-
-        with Connection(QUEUE_BROKER) as connection:
-            producer = connection.Producer()
-            producer.publish(
-                {'message': random_message, 'test': True},
-                exchange=self.exchange,
-                routing_key=self.routing_key,
-                declare=[self.queue],
-            )
+	exchange_name = getenv('KOMBU_EXCHANGE', 'testexchange')
```

Comment:
Looks like there's a bunch of unrelated changes in this PR which clutter up the diff. Perhaps end-of-line changes? I'd recommend to always use [git add --patch](https://git-scm.com/docs/git-add#Documentation/git-add.txt---patch) or similar to ensure that commits only contain required changes.

Reply:
Correction: the change seems to be replacing spaces with tabs. Perhaps related to your editor settings?